modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -332,6 +333,13 @@ public void start(@Nullable String cfg) {
             // Deploy all registered watches because all components are ready and have registered their listeners.
             metaStorageMgr.deployWatches();
 
+            baselineMgr.listenBaselineChange(ctx -> {
+                // TODO: Multi invoke supposed to be here https://issues.apache.org/jira/browse/IGNITE-16063
```

Comment:
Multi invoke is already implemented and can be used.

Reply:
This is out of scope of the current task and will be implemented in the https://issues.apache.org/jira/browse/IGNITE-16063